nixos/maintainers/scripts/openstack/openstack-image.nix

```diff
@@ -7,10 +7,10 @@ with lib;
 {
   imports =
     [ ../../../modules/installer/cd-dvd/channel.nix
-      ../../../modules/virtualisation/nova-config.nix
+      ../../../modules/virtualisation/openstack-config.nix
     ];
 
-  system.build.novaImage = import ../../../lib/make-disk-image.nix {
+  system.build.openstackImage = import ../../../lib/make-disk-image.nix {
```

Comment:
@Lassulus this might need a fix in https://github.com/nix-community/nixos-generators too

Reply:
I could do it.
I also plan to send a post on discourse to ask for some help to publish this image on the NixOS download page.